src/components/KInput/KInput.vue

```diff
@@ -175,8 +175,8 @@ const slots = useSlots()
 const attrs = useAttrs()
 
 const isRequired = computed((): boolean => attrs?.required !== undefined && String(attrs?.required) !== 'false')
-const inputId = attrs.id ? String(attrs.id) : useUniqueId()
-const helpTextId = useUniqueId()
+const inputId = attrs.id ? String(attrs.id) : useId()
```

Comment:
computed

Reply:
You could make these computed, but the `useId()` call would need to be made outside of the computed function and then referenced, similar to this:

```typescript
const defaultId = useId()
const inputId = computed((): string => attrs.id ? String(attrs.id) : defaultId)
```

Reply:
Yes, I have an example in my [first comment](https://github.com/Kong/kongponents/pull/2535#discussion_r1879195321).

Reply:
Ah yep I missed that one 👍🏼

Reply:
Fixed